<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDB</class>
 <widget class="QDialog" name="ConnectDB">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>325</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>在轨数据收集与分析系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>skin/images/QFramer.ico</normaloff>skin/images/QFramer.ico</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="cancelButton_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>66</y>
     <width>355</width>
     <height>220</height>
    </rect>
   </property>
   <property name="title">
    <string>配置数据库</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>13</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>MS Access数据库</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oracle数据库</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>13</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>数据源：</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>40</y>
      <width>311</width>
      <height>168</height>
     </rect>
    </property>
    <property name="title">
     <string>ORACLE</string>
    </property>
    <widget class="QLabel" name="textLabel3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数据库名：</string>
     </property>
     <property name="buddy">
      <cstring>editDatabase</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>80</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel5">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>110</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>主机名：</string>
     </property>
     <property name="buddy">
      <cstring>editHostname</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel5_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>140</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>端口：</string>
     </property>
     <property name="buddy">
      <cstring>textLabel5_2</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="editDatabase">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>20</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ORACLE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editUsername">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>50</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>system</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editHostname">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>110</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPassword">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>80</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DACS2015</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPort">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>140</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1521</string>
     </property>
    </widget>
    <zorder>textLabel5_2</zorder>
    <zorder>editPort</zorder>
    <zorder>label_3</zorder>
    <zorder>editUsername</zorder>
    <zorder>editPassword</zorder>
    <zorder>textLabel3</zorder>
    <zorder>editDatabase</zorder>
    <zorder>editHostname</zorder>
    <zorder>textLabel5</zorder>
    <zorder>label_2</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>40</y>
      <width>311</width>
      <height>168</height>
     </rect>
    </property>
    <property name="title">
     <string>ACCESS</string>
    </property>
    <widget class="QLabel" name="textLabel3_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;驱动：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="buddy">
      <cstring>editDatabase</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Microsoft Access Driver (*.mdb, *.accdb)</string>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel3_2">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>37</y>
       <width>69</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数据库文件：</string>
     </property>
     <property name="buddy">
      <cstring>editDatabase</cstring>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>60</y>
       <width>286</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="openButton">
     <property name="geometry">
      <rect>
       <x>233</x>
       <y>120</y>
       <width>74</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset;border-color:rgb(27, 118, 150);border-width: 1px;border-radius: 0px;</string>
     </property>
     <property name="text">
      <string>选择...</string>
     </property>
     <property name="icon">
      <iconset resource="QFramer.qrc">
       <normaloff>:/skin/icons/dark/appbar.folder.open.png</normaloff>:/skin/icons/dark/appbar.folder.open.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <zorder>openButton</zorder>
    <zorder>textLabel3_2</zorder>
    <zorder>textLabel3_3</zorder>
    <zorder>plainTextEdit</zorder>
    <zorder>lineEdit</zorder>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_4</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>293</y>
     <width>74</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;border-color:rgb(27, 118, 150);border-width: 1px;border-radius: 0px;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="icon">
    <iconset resource="QFramer.qrc">
     <normaloff>:/skin/icons/dark/appbar.close.png</normaloff>:/skin/icons/dark/appbar.close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>293</y>
     <width>74</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;border-color:rgb(27, 118, 150);border-width: 1px;border-radius: 0px;</string>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
   <property name="icon">
    <iconset resource="QFramer.qrc">
     <normaloff>:/skin/icons/dark/appbar.transit.connection.png</normaloff>:/skin/icons/dark/appbar.transit.connection.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>61</height>
    </rect>
   </property>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>0</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="QFramer.qrc">
      <normaloff>:/skin/icons/dark/appbar.close.png</normaloff>:/skin/icons/dark/appbar.close.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>在轨数据收集与分析系统</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>closeButton</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>90</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>用户登录</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel4">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>41</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel4_2">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>90</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>61</red>
          <green>148</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>61</red>
          <green>148</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>61</red>
          <green>148</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;border-color:rgb(27, 118, 150);border-width: 1px;border-radius: 0px;</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>61</red>
         <green>148</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>61</red>
         <green>148</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>61</red>
         <green>148</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>widget</zorder>
  <zorder>cancelButton_2</zorder>
  <zorder>cancelButton</zorder>
  <zorder>okButton</zorder>
  <zorder>exitButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>loginButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>exitButton</tabstop>
  <tabstop>editDatabase</tabstop>
  <tabstop>editUsername</tabstop>
  <tabstop>editHostname</tabstop>
  <tabstop>editPassword</tabstop>
  <tabstop>editPort</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>cancelButton_2</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources>
  <include location="QFramer.qrc"/>
 </resources>
 <connections/>
</ui>
